digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/ProjectDAO.scala

```diff
@@ -173,6 +173,11 @@ object ProjectDAO {
     }
   }
 
+  def updateMValues(projectLink: ProjectLink): Unit = {
+      sqlu"""update project_link set start_addr_m = ${projectLink.startAddrMValue}, end_addr_m = ${projectLink.endAddrMValue} where id = ${projectLink.id}
+          """.execute
+        sqlu"""update lrm_position set start_measure = ${projectLink.startMValue}, end_measure = ${projectLink.endMValue}  where id = ${projectLink.lrmPositionId}""".execute
```

Comment:
I feel its unnecessary to update  START_MEASURE and END_MEASURE of linear location.